### DISCUSSION TITLE
Does TTS Model need Validation set and test set?

### DISCUSSION THREAD

**Discussion by jaehyoun** [coqui-ai/TTS#1930] [General Q&A]
While experimenting, I wasn't thinking about validation and test set.
Yet recently, I read some TTS papers(VITS, Glow-TTS) and found that they usually divide dataset for validation and test set.

What is the usage of validation and test set for TTS?

**Comment by p0p4k** [comment]
Validation set is training-time inference to verify if your model is training as per your needs.
Testing set is testing-time inference data to verify if the model has generalized as per your satisfaction.
